docs/reference/search/multi-search.asciidoc

```diff
@@ -27,53 +37,219 @@ body\n
+example, a default request hits all indices in a cluster. This could cause shard
+request rejections if the number of shards per node is high.
+
+In certain scenarios, parallelism isn't achieved through concurrent requests. In
+those cases, a low value in this  parameter could result in poor performance.
+For example, in an environment where a very low number of concurrent search
+requests are expected, a higher value in this parameter may improve performance.
+--
+
+`pre_filter_shard_size`::
+(Optional, integer)
+Defines a threshold that enforces a pre-filter roundtrip to prefilter search
+shards based on query rewriting if the number of shards the search request
+expands to exceeds the threshold. This filter roundtrip can limit the number of
+shards significantly if for instance a shard can not match any documents based
+on it's rewrite method i.e., if date filters are mandatory to match but the
+shard bounds and the query are disjoint. Defaults to `128`.
+
+`rest_total_hits_as_int`::
+(Optional, boolean)
+If `true`, `hits.total` are returned as an integer in the
+response. Defaults to `false`, which returns an object.
+
+`routing`::
+(Optional, string)
+Custom <<mapping-routing-field,routing value>> used to route search operations
+to a specific shard.
+
+`search_type`::
++
+--
+(Optional, string)
+Indicates whether global term and document frequencies should be used when
+scoring returned documents. 
+
+Options are:
+
+`query_then_fetch`::
+(default)
+Documents are scored using local term and document frequencies for the shard.
+This is usually faster but less accurate.
+
+`dfs_query_then_fetch`::
+Documents are scored using global term and document frequencies across all
+shards. This is usually slower but more accurate.
+--
   
-`query`::
-  (Optional, <<query-dsl,query object>>) Defines the search definition using the 
-  <<query-dsl,Query DSL>>.
-
-include::{docdir}/rest-api/common-parms.asciidoc[tag=routing]
-
-include::{docdir}/rest-api/common-parms.asciidoc[tag=search_type]
+`typed_keys`::
+(Optional, boolean)
+Specifies whether aggregation and suggester names should be prefixed by their
+respective types in the response.
 
-`size`:: 
-  (Optional, integer) The number of hits to return. Defaults to `10`.
+[[search-multi-search-api-request-body]]
+==== {api-request-body-title}
 
+The request body contains a newline-delimited list of search `<header>` and
+search `<body>` objects.
+
+`<header>`::
++
+--
+(Required, object)
+Contains parameters used to limit or change the subsequent search body request.
+
+This object is required for each search body but can be empty (`{}`) or a blank
+line.
+--
+
+`allow_no_indices`:::
+(Optional, boolean)
+If `true`, the request does *not* return an error if a wildcard expression or
+`_all` value retrieves only missing or closed indices.
++
+This parameter also applies to <<indices-aliases,index aliases>> that point to a
+missing or closed index.
+
+`expand_wildcards`:::
```

Comment:
Can this reuse the `common-parms` include as done for the request parameter above?